src/render.js

```diff
@@ -11,8 +11,13 @@ const toggleCell = (cell, v) => cell.className = v ? 'on' : ''
 const updateCell = curry((board, r, c, v) => toggleCell(cell(board, r, c), v))
 
 // lodash/fp "caps" iteratee methods at 1 arg unless this is turned off
+
+// Ugh, this bothers me. I get it, but it's annoying. (I've opened an issue
+// about this and he closed it, heh)
 const eachWithI = each.convert({ cap: false })
 
+// It seems odd to flatten and then reverse engineer row and col. There's
+// probably a nested each way of doing this cleanly.
```

Comment:
I think this came out of an aborted attempt to flatten the state into a 1d array instead of a 2d, in an attempt to reduce the number of object allocations per frame.